Riot/ViewController/ReadReceiptsViewController.h

```diff
@@ -19,6 +19,6 @@
 
 @interface ReadReceiptsViewController : UIViewController
 
-+ (void)openInViewController:(UIViewController *)viewController withRestClient:(MXRestClient *)restClient withRoomMembers:(NSArray <MXRoomMember *> *)roomMembers placeholders:(NSArray <UIImage *> *)placeholders receiptDescriptions:(NSArray <NSString *> *)receiptDescriptions;
++ (void)openInViewController:(UIViewController *)viewController withRestClient:(MXRestClient *)restClient session:(MXSession *)session withRoomMembers:(NSArray <MXRoomMember *> *)roomMembers placeholders:(NSArray <UIImage *> *)placeholders receipts:(NSArray <MXReceiptData *> *)receipts;
```

Comment:
I would prefer split this operation in 2:
- one to create and return a new `ReadReceiptsViewController ` object:
`+ (instancetype)readReceiptsViewControllerBasedOnReceiptSendersContainer:(MXKReceiptSendersContainer*)receiptSendersContainer andSession:(MXSession *)session;`

Note: the REST client will be deduced from the provided session. The receiptSendersContainer instance will provide all the other inputs.
   
- another to handle its display: `- (void)showInViewController:(UIViewController*)viewController;`